src/routes/(site)/genuary/media.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+  export let video: boolean;
+  export let src: string;
+  export let view = '';
+  export let alt = '';
+  export let rounded = false;
+</script>
+
+<a href={src} class="media-container" target="_blank" rel="noopener noreferrer">
+  <svelte:element
+    this={video ? 'video' : 'img'}
+    {src}
+    {alt}
+    class="media"
+    class:wide={view === 'wide'}
+    class:tall={view === 'tall'}
+    class:big={view === 'big'}
+    loading="lazy"
+    autoplay={video}
+    loop={video}
+    muted={true}
```

Comment:
The videos are not muted despite setting the `muted` attribute to `true`.

Reply:
Possibly this Codepen is doing something we aren’t. 

https://codepen.io/simevidas/pen/PJMENp